services/alerts/src/useAlert.ts

```diff
@@ -17,15 +18,25 @@ export const useAlert = (
             const resolvedOptions =
                 typeof options === 'function' ? options(props) : options
 
-            alertRef.current = add(
-                {
-                    message: resolvedMessage,
-                    options: resolvedOptions,
-                },
-                alertRef
-            )
+            if (plugin && parentAlertsAdd && !showAlertsInPlugin) {
```

Comment:
I passed this showAlertsInPlugin down to useAlert as I thought it made the logic clearer here, but it could be incorporated into logic further up and then not passed down.